src/ontology/edits/oeo-social.omn

```diff
@@ -213,7 +213,7 @@ ObjectProperty: OEO_00020056
 ObjectProperty: OEO_00020070
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Is traded at is a relation that holds between a market exchange and the financial instruments, commodities, or other products, services, or goods which are traded there.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Is traded at is a relation that holds between a market exchange and commodities which are traded there."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Is traded at is a relation that holds between a commodity and a market exchange."@en,
```

The definitions sound like market exchanges are traded at commodities and vice versa. I'd prefer to firstly mention the domain and secondly the range. 
The second part is just a duplication without further specification. I suggest to either skip it completely or add something like _... where the commodity is offered for trade/sale at the market exchange._

Reply:
I agree with the change including the extension: _... where the commodity is offered for trade/sale at the market exchange._